components/src/main/kotlin/com/mineinabyss/geary/ecs/helper/ComponentsHelper.kt

```diff
@@ -0,0 +1,8 @@
+package com.mineinabyss.geary.ecs.helper
+
+import com.mineinabyss.geary.ecs.api.GearyComponent
+import com.mineinabyss.geary.ecs.serialization.Formats
+import kotlin.reflect.KClass
+
+public fun mapComponentNamesToClasses(names: Collection<String>): Collection<KClass<out GearyComponent>> =
```

Comment:
Small thing but an extension function would be best here I think (i.e. `Colleciton<String>.toComponentClasses()`).

Try using IntelliJ's function refactor menu (`Ctrl + F6`) to do this, it's a huuuuge time-saver.

Reply:
✔️